Cargo.toml

```diff
@@ -27,8 +27,8 @@ dexter-stable-pool   = "1.1.1"
 dexter-weighted-pool = "1.1.1"
 dexter-lp-token  = "1.0.0"
 dexter-router    = "1.1.0"
-cosmwasm-schema  = "1.5.0"
-cosmwasm-std     = { version = "1.5", features = ["stargate"] }
+cosmwasm-schema  = "1.5.4"
+cosmwasm-std     = { version = "1.5.4", features = ["stargate", "cosmwasm_1_2"] }
```

Comment:
@albertandrejev Does this break our contracts being able to be deployed on chains without cosmwasm 1.2?

Reply:
@albertandrejev why do we need it, can we remove?

Reply:
let me check it

Reply:
removed

Reply:
Awesome, thanks